src/cloudsc_gpu/CMakeLists.txt

```diff
@@ -142,7 +189,83 @@ if( HAVE_CLOUDSC_GPU_SCC_K_CACHING )
     )
 endif()
 
+
+if( HAVE_CLOUDSC_GPU_OMP_SCC )
+    
+    list( APPEND CLOUDSC_GPU_OMP_SCC_DEFINITIONS CLOUDSC_GPU_OMP_SCC )
+    if( HAVE_OMP_TARGET_LOOP_CONSTRUCT )
+            list( APPEND CLOUDSC_GPU_OMP_SCC_DEFINITIONS HAVE_OMP_TARGET_LOOP_CONSTRUCT )
+        if( HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_PARALLEL )
+                list( APPEND CLOUDSC_GPU_OMP_SCC_DEFINITIONS HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_PARALLEL )
+        endif()
+        if( HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_THREAD )
+                list( APPEND CLOUDSC_GPU_OMP_SCC_DEFINITIONS HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_THREAD )
+        endif()
+    endif()
+
+    ecbuild_add_executable(
+        TARGET dwarf-cloudsc-gpu-omp-scc
+        SOURCES
+            dwarf_cloudsc_gpu.F90
+            cloudsc_driver_gpu_omp_scc_mod.F90
+            cloudsc_gpu_omp_scc_mod.F90
+        LIBS
+            cloudsc-common-lib
+        DEFINITIONS ${CLOUDSC_DEFINITIONS} ${CLOUDSC_GPU_OMP_SCC_DEFINITIONS}
+    )
+
+    ecbuild_add_test(
+        TARGET dwarf-cloudsc-gpu-omp-scc-serial
+        COMMAND bin/dwarf-cloudsc-gpu-scc-omp
+        ARGS 1 1000 128
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../..
+        OMP 1
+        ENVIRONMENT "NVCOMPILER_ACC_CUDA_HEAPSIZE=64M"
+    )
+endif()
+
+
+if ( HAVE_CLOUDSC_GPU_OMP_SCC_STACK )
+
+    list( APPEND CLOUDSC_GPU_OMP_SCC_STACK_DEFINITIONS CLOUDSC_GPU_OMP_SCC_STACK )
+    if( HAVE_OMP_TARGET_LOOP_CONSTRUCT )
+	    list( APPEND CLOUDSC_GPU_OMP_SCC_STACK_DEFINITIONS HAVE_OMP_TARGET_LOOP_CONSTRUCT )
+        if( HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_PARALLEL )
+		list( APPEND CLOUDSC_GPU_OMP_SCC_STACK_DEFINITIONS HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_PARALLEL )
+        endif()
+        if( HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_THREAD )
+		list( APPEND CLOUDSC_GPU_OMP_SCC_STACK_DEFINITIONS HAVE_OMP_TARGET_LOOP_CONSTRUCT_BIND_THREAD )
+        endif()
+    endif()
+
+    ecbuild_add_executable( TARGET dwarf-cloudsc-gpu-omp-scc-stack
+        SOURCES
+            dwarf_cloudsc_gpu.F90
+            cloudsc_driver_gpu_omp_scc_stack_mod.F90
+            cloudsc_gpu_omp_scc_stack_mod.F90
+        LIBS
+            cloudsc-common-lib
+	    DEFINITIONS ${CLOUDSC_DEFINITIONS} ${CLOUDSC_GPU_OMP_SCC_STACK_DEFINITIONS}
+    )
+
+    if( CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
+        target_compile_options( dwarf-cloudsc-gpu-scc-stack PRIVATE "-fcray-pointer" )
+    elseif( CMAKE_Fortran_COMPILER_ID MATCHES "NVHPC" OR CMAKE_Fortran_COMPILER_ID MATCHES "PGI" )
+        target_compile_options( dwarf-cloudsc-gpu-scc-stack PRIVATE "-Mcray=pointer" )
```

Comment:
Target name needs to be `dwarf-cloudsc-gpu-omp-scc-stack` here. That's causing the Github Actions to fail currently